build/ps-entry.sh

```diff
@@ -424,14 +432,19 @@ if [[ $originalArgOne == mongo* ]]; then
 			# if --config arg is present, try to get tlsMode from it
 			if _parse_config "${mongodHackedArgs[@]}"; then
 				tlsMode=$(jq -r '.net.tls.mode // "preferSSL"' "${jsonConfigFile}")
+
+				if [ -z "$tlsMode" ]; then
+					tlsMode='preferTLS'
+				fi
```

Comment:
why do we need this?

Reply:
Because for some reason `jq -r '.net.tls.mode // "preferSSL"' "${jsonConfigFile}"` returns an empty string even though we have ` // "preferSSL` as a default. I put it for now just to make it work, I'm troubleshooting why jq command does not return properly.

Reply:
Here is the reason: [b1f4180](https://github.com/percona/percona-server-mongodb-operator/pull/1108/commits/b1f4180f4f20a3be20471c169f9b7571255bf9df)